GNUmakefile

```
@@ -17,17 +17,17 @@ install: fmtcheck
 uninstall:
 	@rm -vf $(DIR)/terraform-provider-datadog
 
-test: fmtcheck
-	go test -i $(TEST) || exit 1
+test: get-test-deps fmtcheck
+	gotestsum --format testname -- -i $(TEST) || exit 1
```

This should only install dependencies.
```suggestion
	go test -i $(TEST) || exit 1
```